packages/terra-dropdown-button/tests/jest/DropdownList.test.jsx

```diff
@@ -118,7 +113,7 @@ describe('Dropdown List', () => {
 
   it('should set the aria-label property to empty on keydown on Mac', () => {
     //  Sets the mock return value for isMac
-    SharedUtil.isMac.mockReturnValue(true);
+    jest.spyOn(Util, 'isMac').mockReturnValue(true);
```

Comment:
if you're creating spy then make sure you clear them on `afterAll` OR `afterEach` as a best practice : https://jestjs.io/docs/jest-object#jestspyonobject-methodname

Reply:
Added as suggested.Thanks